sway-core/src/control_flow_analysis/flow_graph/mod.rs

```diff
@@ -206,7 +206,26 @@ impl<'cfg> ControlFlowGraph<'cfg> {
 
         tracing::info!(
             "{:?}",
-            Dot::with_config(&string_graph, &[Config::EdgeNoLabel])
+            Dot::with_attr_getters(
+                &string_graph,
+                &[Config::NodeNoLabel, Config::EdgeNoLabel],
+                &|_, er| format!("label = {:?}", er.weight()),
+                &|_, nr| {
+                    let node = &self.graph[nr.0];
+                    let mut shape = "";
+                    if self.entry_points.contains(&nr.0) {
+                        shape = "shape=doubleoctagon";
+                    }
+                    let mut url = "".to_string();
+                    if let Some(span) = node.span() {
+                        if let Some(path) = span.path_str() {
+                            let (line, col) = span.start_pos().line_col();
+                            url = format!("URL = \"vscode://file/{path}:{line}:{col}\"");
```

Comment:
Because not everyone uses VSCode, baking this into the code directly is a bit odd. However, we could amend the flag `--dca-graph` to accept a graph format or something similar so that people who use VSCode can then use the new flag (or maybe pass a value for it as in `--dca-graph vscode-url` to get the VSCode URL.

Reply:
Added a new flag, we can now call forc with the argument `--dca-graph-url-format`.
For vscode we would use: `--dca-graph-url-format 'vscode://file/{path}:{line}:{col}'`